Activities/Tests/Python/while true.py

```diff
@@ -0,0 +1,2 @@
+while True:
```

Comment:
is best not to have space in the python name

Reply:
Hello. thank you for the review. This file will be removed for now as it is part of another test.